db/migrate/20220721201913_create_assets.rb

```diff
@@ -0,0 +1,15 @@
+class CreateAssets < ActiveRecord::Migration[7.0]
+  def change
+    create_table :assets do |t|
+      t.date :approximate_purchase_date, null: false
+      t.integer :asset_type, null: false, default: 0
```

Comment:
```suggestion
      t.string :asset_type, null: false, default: "laptop"
```

(Related to my above comment)